ckan/requirements.txt

```diff
@@ -13,7 +13,7 @@ chardet==3.0.4
 -e git+https://github.com/ckan/ckan.git@e5105334fbdfd312412da4e05221ce0588dc2dfd#egg=ckan
 -e git+https://github.com/ckan/ckanext-archiver.git@035728ddd739c5029f89e65fcb1d9048934c9ed9#egg=ckanext-archiver
 -e git+https://github.com/GSA/ckanext-datagovcatalog.git@64e65702ae1eb5e46d9f37139dc4044c0f253526#egg=ckanext-datagovcatalog
--e git+https://github.com/GSA/ckanext-datagovtheme.git@c9d80fc05039fb4021c90e24eab2d446aac5f8b3#egg=ckanext-datagovtheme
+-e git+https://github.com/GSA/ckanext-datagovtheme.git@704c45921a9246fd6072ea99711992e57d3bc416#egg=ckanext-datagovtheme
```

Comment:
Why did datagovtheme also change?

Reply:
datagovtheme is pointed to main branch, not commit pinned. Anytime we run `make update-dependencies` it will grab latest from datagovtheme main.